torchvision/prototype/transforms/_augment.py

```diff
@@ -129,14 +129,15 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         lam = params["lam"]
-        if isinstance(inpt, features.Image) or features.is_simple_tensor(inpt):
-            if inpt.ndim < 4:
-                raise ValueError("Need a batch of images")
+        if isinstance(inpt, (features.Image, features.Video)) or features.is_simple_tensor(inpt):
+            expected_dim = 5 if isinstance(inpt, features.Video) else 4
+            if inpt.ndim != expected_dim:
```

Comment:
You mean batched videos, i.e. `(B, T, C, H, W)`, right? Because

> videos but in Tensor format

would be `(T, C, H, W)` and that is already covered. Although I'm not sure if we want to apply this to the temporal dimension.